firestore/cloud-client/distributed_counters_test.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import distributed_counters
+
+
+def test_distributed_counters(db):
+    doc_ref = db.collection("counter_samples").document("distributed_counter")
```

Comment:
While this technically passed our checks, it's because this function never was executed. Pytest picks up tests by searching for the _suffix_ `_test`, **not** the _prefix_ `test_` :)

That being said, this should fail once executed as there's no reference to `db` mentioned anywhere. One way to do this might be to create a `pytest.fixture` function that a) allows you to create a firestore client that you can reference and b) allows you to move your clean up steps out of the function.

Reply:
I don't think using suffix is right, unless there are some internal preferences set. Here is how I'm running the test:
![image](https://user-images.githubusercontent.com/26030071/69138901-93f4f100-0ad0-11ea-84e6-06343a5f798a.png)

But if I'll move `test` from prefix to suffix, Pytest won't find any tests in this file.
![image](https://user-images.githubusercontent.com/26030071/69139051-d1597e80-0ad0-11ea-9ec5-8ecbd6f701bd.png)
I suspect, autochecks don't run tests session at all, that's why they are always passing successfully.

Reply:
Ahh yes, my apologies you're totally right, prefix is correct.